pkgs/applications/office/timetable/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, gtk3
+, hicolor-icon-theme
+, json-glib
+, libgee
+, meson
+, ninja
+, pkgconfig
+, pantheon
+, python3
+, wrapGAppsHook }:
+
+
+stdenv.mkDerivation rec {
+  pname = "timetable";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "lainsce";
+    repo = "timetable";
+    rev = version;
+    sha256 = "080xgp917v6j40qxy0y1iycz01yylbcr8pahx6zd6mpi022ccfv0";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    pantheon.vala
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+    hicolor-icon-theme
+    json-glib
+    libgee
+    pantheon.granite
+  ];
+
+  postPatch = ''
+    chmod +x meson/post_install.py
+    patchShebangs meson/post_install.py
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Plot out your own timetable for the week and organize it";
+    homepage = https://github.com/lainsce/timetable;
```

Comment:
Can you please use quotes?

Reply:
I'm not sure to understand what do you mean by _quotes_ ?

Reply:
```
homepage = "https://github.com/lainsce/timetable";
```

Reply:
hmm I'm pretty sure any other package doesn't use quotes for the homepage link :thinking:

Reply:
While this is true that older packages are not using quotes, this is being discussed as a new policy (https://github.com/NixOS/rfcs/pull/45). You don't have to of course, but it would help in the future :)

Reply:
Okay, I see the idea. I wasn't aware of that discussion, thanks :)

I've updated the commit with quotes